### DISCUSSION TITLE
All entities unavailable - logs show proper connection

### DISCUSSION THREAD

**Discussion by st0n1-boop** [moralmunky/Home-Assistant-Mail-And-Packages#980] [Q&A]
Hi all,

I'm trying to set this up, but am not able to make headway.
It seems that a connection to the imap server (GMX) can be made successfully, and the logs show that amazon packages are being counted. However, all entities are greyed out and show as unavailable.

![image](https://github.com/user-attachments/assets/556192e7-c014-4e47-a2cc-b7c2d0286592)

![image](https://github.com/user-attachments/assets/db890ea7-1f14-4100-ad5a-da12a0d4d78f)

Any idea where I'm going wrong?

Thanks!

**Comment by firstof9** [comment]
Look for a message in your log that says "timeout", if you see that increase the IMAP timeout in the configuration.
